example/65_gemm_multiply_multiply/gemm_multiply_multiply_xdl_fp8_bpreshuffle.cpp

```diff
@@ -324,7 +324,9 @@ int main(int argc, char* argv[])
                                KBatch,
                                a_element_op,
                                b_element_op,
-                               cde_element_op);
+                               cde_element_op,
+                               N,
+                               K);
```

Comment:
Why passed another set of NK?

Reply:
for N and K padding, if N or K padded, must passed the value.

Reply:
Tools like ckprofiler that pass in data without padding, but still detect instances with NKpadding, may crash

Reply:
done